src/Mod/Arch/Resources/ui/preferences-ifc.ui

```diff
@@ -533,7 +566,8 @@
       <item>
        <widget class="Gui::PrefCheckBox" name="checkBox_19">
         <property name="toolTip">
-         <string>When exporting an IFC file, if no site is found in the FreeCAD document, a default one will be added. A site is not mandatory by the IFC standard, but it is a common practice to always have at least one in the file.</string>
+         <string>If no site is found in the FreeCAD document, a default one will be added.
+A site is not mandatory but a common practice to have at least one in the file.</string>
```

Comment:
A site is not mandatory but a common practice __is__ to have at least one in the file

Reply:
I am not a native speaker, I thought I could omit a second 'is'